### PR TITLE
Auth 36 navbar

### PR DESCRIPTION
**Pull request by mikexd-dev**
# Pull Request Overview

### Description:

> _Write a short description for this PR_

- Reimplemented the Navbar component

### Key Changes:

> _List out key changes made in this PR_

- navbar contain search icon and connect button (disabled for now)
- search icon opens a modal similar to tailwind doc design
- navbar cater for responsiveness
- use drawer for mobile menu instead to cater for a bigger menu options in the future

<br>

---

### Link to story:

https://menyala.atlassian.net/browse/AUTH-36

### CODE REVIEW

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -5,7 +5,9 @@ name: "Chromatic"
 
 # Event for the workflow
 on: push
-
+ push:
```
Comment (terrytjw) [inline .github/workflows/chromatic.yml:8]: are we still keeping this file?

File: components/Navbar.tsx [hunk 2]
```diff
@@ -1,7 +1,150 @@
...
+        strokeLinecap="round"
+        strokeLinejoin="round"
+        strokeWidth="2"
+        d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
+      />
+    </svg>
+  );
+}
+
+function MenuIcon(props: JSX.IntrinsicAttributes & SVGProps<SVGSVGElement>) {
```
Comment (terrytjw) [inline components/Navbar.tsx:30]: change to const

File: components/Navbar.tsx [hunk 3]
```diff
@@ -1,7 +1,150 @@
...
+import Link from "next/link";
+import Image from "next/image";
 
-const Navbar = () => {
-  return <div>Navbar</div>;
+type NavbarProps = {
+  children: ReactNode;
+};
+
+function SearchIcon(props: JSX.IntrinsicAttributes & SVGProps<SVGSVGElement>) {
```
Comment (terrytjw) [inline components/Navbar.tsx:10]: change to const syntax

File: components/Navbar.tsx [hunk 4]
```diff
@@ -1,7 +1,150 @@
 import React from "react";
+import { ReactNode, SVGProps } from "react";
+import Link from "next/link";
+import Image from "next/image";
 
-const Navbar = () => {
-  return <div>Navbar</div>;
+type NavbarProps = {
```
Comment (terrytjw) [inline components/Navbar.tsx:6]: Place the `NavbarProps` directly above its functional component

**Review by terrytjw — CHANGES_REQUESTED** [review]
just some small syntax changes, open for discussion tomorrow morning

File: components/Navbar.tsx (hunk 2 above)
Comment (mikexd-dev) [reply components/Navbar.tsx]: Updated in utils/svgs/Menu.tsx

File: components/Navbar.tsx (hunk 3 above)
Comment (mikexd-dev) [reply components/Navbar.tsx]: Updated in utils/svgs/Search.tsx

**Review by terrytjw — APPROVED** [review]

**Pull request merged**